department-of-veterans-affairs/vets-api
# Fix pensions spouseIsVeteran

## Summary

When the veteran's marital status was set to Separated, the "spouseIsVeteran" field wasn't correctly populated in the PDF. 

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/75004

## Testing done


Steps:

-  Start a new Pension application
- Enter minimal data until you get to the marriage section. Select "Separated" for the marital status
- Enter spouse's into making sure to select YES for the question "Is your spouse a veteran?"
continue with minimal data
- Submit the form and inspect the generated PDF

Before this PR: In the PDF, NO is selected for this question
New result: In the PDF, YES should be selected

## Screenshots

Should populate field if separated:
<img width="982" alt="Screenshot 2024-01-31 at 11 11 45 AM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/21046714/7ae98753-11a9-4404-a5b1-2fa10027f05e">
Should populate field if married:
<img width="967" alt="Screenshot 2024-01-31 at 11 15 26 AM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/21046714/8c7f4495-1711-493f-a951-02eb7e3983ee">
Should not populate field for other marital statuses:
<img width="963" alt="Screenshot 2024-01-31 at 11 16 50 AM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/21046714/7c4c7b69-833f-441c-aef2-2d287a651f81">

## What areas of the site does it impact?
Benefits Pensions

## Acceptance criteria
